scripts/TestMaliciousIP.java

```diff
@@ -0,0 +1,79 @@
+package com.google.netpcapanalysis.lookup_ip;
+
+import com.google.netpcapanalysis.dao.MaliciousIPDaoImpl;
+import java.util.ArrayList;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.BufferedReader;
+import com.google.netpcapanalysis.mockdata.MockDataLoader;
+import com.google.netpcapanalysis.models.PCAPdata;
+import com.google.netpcapanalysis.interfaces.dao.MaliciousIPDao;
+
+
+public class TestMaliciousIP {
+  private String CSV_FILE = "maliciousIPTest.csv"; //first 1-30 valid IP's, 31-60 malicious IP's
+  ArrayList<PCAPdata> IPlist;
+    long AvgTime;
+    long TotalTime;
+    int flagged_true;
+    int flagged_false;
+
+  public TestMaliciousIP(){
+        MockDataLoader mockData = new MockDataLoader();
+        IPlist = mockData.CSVDataLoader(CSV_FILE);
+        flagged_true=0;
+        flagged_false=0;
+        AvgTime = 0L;
+        TotalTime = 0L;
+    }
+
+   public void test(){
```

Comment:
This is good but do not run this as part of CI testing . You could also put this in a another directory like scripts since this is more like a Benchmark than a test.